libcudacxx/include/cuda/std/detail/libcxx/include/complex

```diff
@@ -292,6 +292,21 @@ template<class T> complex<T> tanh (const complex<T>&);
 
 _LIBCUDACXX_BEGIN_NAMESPACE_STD
 
+template<class _Tp>
+struct __is_complex_float {
```

Comment:
Would it make sense to rename this to `__is_extended_floating_point` and move it to `is_floating_point.h`?

I would imagine there might be other places we will want to differentiate